test/unit/controllers/declaration/SummaryControllerSpec.scala

```diff
@@ -134,7 +131,11 @@ class SummaryControllerSpec extends ControllerSpec with ErrorHandlerMocks with O
         session(result).get(ExportsSessionKeys.declarationId) must be(None)
         redirectLocation(result) must be(Some(controllers.declaration.routes.ConfirmationController.displayPage().url))
         flash(result).get("LRN") must be(Some("123LRN"))
-        verify(mockSubmissionService).submit(any[ExportsDeclaration])(any[JourneyRequest[_]], any[HeaderCarrier], any[ExecutionContext])
+        verify(mockSubmissionService).submit(any[ExportsDeclaration])(
+          any[JourneyRequest[_]],
```

Comment:
In general I'm thinking that it's good to add those types for any. I mean `JourneyRequest[_]` etc, there is no option to invoke it with different type there. Just any() will be shorter, but I can be wrong about it.